app/models/experimentation/experiment.rb

```diff
@@ -13,9 +13,15 @@ class Experiment < ActiveRecord::Base
       live: "live",
       complete: "complete"
     }, _prefix: true
-    enum experiment_type: {
-      current_patient_reminder: "current_patient_reminder",
-      stale_patient_reminder: "stale_patient_reminder"
-    }, _prefix: true
```

Comment:
Dropping the `experiment_type` and replacing it with `fetch_message` and `queueable_message_exists?` below. Since we're modelling and recording treatment groups and membership, these can just be looked up as necessary.